app/controllers/ems_common.rb

```diff
@@ -1005,6 +1006,7 @@ def process_emss(emss, task)
           :target_class => model.to_s)
     elsif task == "destroy"
       model.where(:id => emss).order("lower(name)").each do |ems|
+        ems.disable_ems
```

Comment:
> How about addressing the initial cause for having this disabled flag first?
We would still have a disabled flag, but focus on the scheduled deletion and pre disablement of a manager first?

@durandom Would this be enough? disabling the ems here before a provider delete is queued?